src/ui/public/state_management/state_monitor.js

```diff
@@ -50,9 +55,22 @@ function stateMonitor(state, defaultState) {
   };
 
   return {
+    setDefaultState(customInitialState) {
```

Comment:
Sorry, I don't think I was clear. I think if we're going to use a term like `initial`, `original` or `default`, it should be used consistently in all variable and method names.

* `setDefaultState` -> `setInitialState`
* `setOriginalState` -> `setInitialState`
* `originalState` -> `initialState`